File: iact-check-core/src/main/kotlin/ch/iact/iactcheck/dto/CustomerBrandingDTO.kt
package ch.iact.iactcheck.dto

data class CustomerBrandingDTO(
    val id: Long,
    val customerId: Long,
    val primaryColour: String,
    val backgroundColour: String,
    val accentColour: String,
    val textColour: String,
    val font: String,
    val theme: String
)

File: iact-check-webapp/src/main/resources/webapp/src/app/shared/dtos/customer-branding-dto.ts
import {Theme} from "../model/theme";

export interface CustomerBrandingDTO {
  id: number;
  customerId: number;
  primaryColour: string;
  backgroundColour: string;
  accentColour: string;
  textColour: string;
  font: string;
  theme: Theme;
}
